src/main/java/com/minecolonies/coremod/achievements/ModAchievements.java

```diff
@@ -39,279 +38,267 @@
-                                                                                              achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementBuildingMiner   = new MineColoniesAchievement("upgrade.miner.first", 2, 8, ModBlocks.blockHutMiner, achievementTownhall);
     /**
      * Max out a miner.
      */
-    public static final Achievement achievementUpgradeMinerMax = new MineColoniesAchievement("upgrade.miner.max", 4, 8, ModBlocks.blockHutMiner,
-                                                                                              achievementBuildingMiner).registerStat();
+    public static final MineColoniesAchievement achievementUpgradeMinerMax = new MineColoniesAchievement("upgrade.miner.max", 4, 8, ModBlocks.blockHutMiner, achievementBuildingMiner);
 
     /**
      * Upgrade a fisher to lv 1.
      */
-    public static final Achievement achievementBuildingFisher   = new MineColoniesAchievement("upgrade.fisher.first", 2, 4, ModBlocks.blockHutFisherman,
-                                                                                               achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementBuildingFisher   = new MineColoniesAchievement("upgrade.fisher.first", 2, 4, ModBlocks.blockHutFisherman, achievementTownhall);
     /**
      * Max out a fisher.
      */
-    public static final Achievement achievementUpgradeFisherMax = new MineColoniesAchievement("upgrade.fisher.max", 4, 4, ModBlocks.blockHutFisherman,
-                                                                                               achievementBuildingFisher).registerStat();
+    public static final MineColoniesAchievement achievementUpgradeFisherMax = new MineColoniesAchievement("upgrade.fisher.max", 4, 4, ModBlocks.blockHutFisherman, achievementBuildingFisher);
 
     /**
      * Upgrade a farmer to lv 1.
      */
-    public static final Achievement achievementBuildingFarmer   = new MineColoniesAchievement("upgrade.farmer.first", 2, 14, ModBlocks.blockHutFarmer,
-                                                                                               achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementBuildingFarmer   = new MineColoniesAchievement("upgrade.farmer.first", 2, 14, ModBlocks.blockHutFarmer, achievementTownhall);
     /**
      * Max out a farmer.
      */
-    public static final Achievement achievementUpgradeFarmerMax = new MineColoniesAchievement("upgrade.farmer.max", 4, 14, ModBlocks.blockHutFarmer,
-                                                                                               achievementBuildingFarmer).registerStat();
+    public static final MineColoniesAchievement achievementUpgradeFarmerMax = new MineColoniesAchievement("upgrade.farmer.max", 4, 14, ModBlocks.blockHutFarmer, achievementBuildingFarmer);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 188 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)